runners/pvc_autoresizer.go

```diff
@@ -240,8 +245,8 @@ func convertSizeInBytes(valStr string, capacity int64, defaultVal string) (int64
 		if err != nil {
 			return 0, err
 		}
-		if rate < 0.0 || 100.0 < rate {
-			return 0, fmt.Errorf("annotation value should be between 0%% to 100%%: %s", valStr)
+		if rate < 0 {
+			return 0, fmt.Errorf("annotation value should be positive: %s", valStr)
 		}
 
 		// rounding up the result to Gi
```

Comment:
misleading

Reply:
What is misleading?

Reply:
The comment "rounding up the result to Gi" does not describe the real behavior of the following code.
This segment is not modified in this PR, though.

Reply:
Okay, I will update it as well as README.